README.md

```diff
@@ -506,10 +503,10 @@ const count = 2 // it tries to run 2(), but 2 is not a function
 
 ```javascript
 // for global variables names we use the const/let keyword and UPPER_SNAKE_CASE
-let MUTABLE_GLOBAL = "mutable value"
```

Comment:
The file went through many formatting changes that are not under the scope of this PR